index.d.ts

```diff
@@ -1,4 +1,5 @@
 export interface FastifyError extends Error {
```

Comment:
this needs to be a class

Reply:
An interface with a new() should be the same as a class.

Reply:
Ok, I double checked this: it should work only when also a var declared using the same name as interface that represents the actual instance. It is better and more simple to use class in this case. :)